static/app/components/events/interfaces/breadcrumbs/breadcrumb/icon.tsx

```diff
@@ -1,23 +1,23 @@
-import * as React from 'react';
+import {memo} from 'react';
 
 import SvgIcon from 'app/icons/svgIcon';
 
 type SvgIconProps = React.ComponentProps<typeof SvgIcon>;
 
 import {BreadcrumbsWithDetails} from 'app/types/breadcrumbs';
 
-import {IconWrapper} from './styles';
+import {IconWrapper} from '../styles';
 
 type Props = Pick<BreadcrumbsWithDetails[0], 'color' | 'icon'> &
   Pick<SvgIconProps, 'size'>;
 
-const Icon = React.memo(({icon, color, size}: Props) => {
+function Icon({icon, color, size}: Props) {
   const Svg = icon as React.ComponentType<SvgIconProps>;
   return (
     <IconWrapper color={color}>
       <Svg size={size} />
     </IconWrapper>
   );
-});
+}
 
-export default Icon;
+export default memo(Icon);
```

Comment:
Hmm we should probably add an eslint rule to not allow anonymous default exports. This can make it harder to debug.

Can we do something like this?

```javascript
const Icon = memo(function Icon({icon, color, size}: Props) {
```

Reply:
totally agree.. will check if we can this rule